src/main/java/cz/mzk/utils/FileWriterUtils.java

```diff
@@ -0,0 +1,45 @@
+package cz.mzk.utils;
+
+import lombok.extern.slf4j.Slf4j;
+
+import java.io.*;
+import java.nio.file.Paths;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+@Slf4j
+public class FileWriterUtils {
+
+    private final String fileName;
+    private final String logsDir;
+
+    public FileWriterUtils(final String fileName) {
+        this.fileName = fileName;
+        this.logsDir = getWorkDirLogs();
+    }
+
+    private String getWorkDirLogs() {
+        final String projectDir = Paths.get("").toAbsolutePath().toString();
+        return projectDir + "/logs/";
+    }
+
+    private String getTextFileNameWithDate() {
+        final SimpleDateFormat simpleDate = new SimpleDateFormat("_dd-MM-yyyy-HH:mm:ss");
+        return this.fileName + simpleDate.format(new Date()) + ".txt";
+    }
+
+    public void writeLine(final String rootUuid) {
+        try {
+            final String fileName = this.logsDir + getTextFileNameWithDate();
+            final FileWriter fileWriter = new FileWriter(fileName, true);
+            final BufferedWriter bufferedWriter = new BufferedWriter(fileWriter);
+            bufferedWriter.write(rootUuid);
+            bufferedWriter.newLine();
+            bufferedWriter.close();
```

Comment:
pokazde kdyz je potreba zapsat radek, oteviras soubor, zapisujes a zaviras
je to spatne, IO operace jsou drahe a neni tento postup optimalni
BufferedWritter klidne muze zustat atributem FileWriterUtils po otevreni, po zpracovani skriptem proste uzavres zdroje